documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -56,41 +58,44 @@ Cruise Control does not perform inter-broker and intra-broker balancing at the s
 Cruise Control provides several configuration options for tuning the rebalance parameters discussed above.
 You can set these tuning options at either the xref:ref-cruise-control-configuration-{context}[Cruise Control server] or xref:proc-generating-optimization-proposals-{context}[optimization proposal] levels:
 
-* The Cruise Control server setting can be set in the Kafka custom resource under `Kafka.spec.cruiseControl.config`. 
-* The individual rebalance performance configurations can be set under `KafkaRebalance.spec`. 
-
-The relevant configurations are summarized below:
+* The Cruise Control server setting can be set in the Kafka custom resource under `Kafka.spec.cruiseControl.config`.
+* The individual rebalance performance configurations can be set under `KafkaRebalance.spec`.
 
-|============================================================================================================================
-| Server and `KafkaRebalance` Configuration                | Description                                          | Default Value
+The relevant configurations are summarized in the following table.
 
-| `num.concurrent.partition.movements.per.broker`   .2+| 
-  The maximum number of inter-broker partition movements in each partition reassignment batch              .2+| 5 
-| `concurrentPartitionMovementsPerBroker` 
+.Rebalance performance tuning configuration
```

Comment:
This table seems to be wrong:
* I don't think it makes it really clear what is what. I think that using 4 columns (One for the configuration key in `Kafka.spec.cruiseControl.config`, one for `KafkaRebalance.spec`, one for description and one for default value) would be much better. If the problem is that it would be too wide, maybe it should be split into two separate tables. The problem of the current table design shows for example with the `rebalanceDisk` row where it is absolutely not clear where it belongs-
* Your changes of the `KafkaRebalance.spec` fields seem wrong - they are not supported by the [API](https://strimzi.io/docs/operators/latest/full/configuring.html#type-KafkaRebalanceSpec-reference)

Reply:
I've changed to four columns, switched them around and modified the titles
I've changed the KafkaBalance properties